sdk/dotnet/go.mod

```diff
@@ -1,3 +1,3 @@
 module fake_dotnet_module // Exclude this directory from Go tools
 
-go 1.19
+go 1.17
```

Comment:
are we  not pinning to `go 1.20.1` ?

Reply:
ah nvm, i see your description in https://github.com/pulumi/pulumi-kafka/pull/142

Reply:
The underlying code generator maintained in pulumi/pkg should update this if they want to, but it's a pseudo-module anyway so it should not be terribly important.